vnext/Microsoft.ReactNative/Pch/pch.h

```diff
@@ -26,6 +26,34 @@
 // When WINAPI_FAMILY is DESKTOP_APP, windows.h creates a macro for GetCurrentTime, which conflicts with other headers
 #undef GetCurrentTime
 
+// Define environment variable RNW_FASTBUILD=true to improve compilation speed on your PC.
+#ifdef RNW_FASTBUILD
+#include <winrt/Windows.ApplicationModel.Activation.h>
+#include <winrt/Windows.Devices.Input.h>
+#include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Windows.Foundation.Metadata.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Graphics.Display.h>
+#include <winrt/Windows.Networking.Sockets.h>
+#include <winrt/Windows.Security.Cryptography.h>
+#include <winrt/Windows.Storage.Streams.h>
+#include <winrt/Windows.System.h>
+#include <winrt/Windows.UI.Composition.h>
+#include <winrt/Windows.UI.ViewManagement.Core.h>
+#include <winrt/Windows.UI.ViewManagement.h>
+#include <winrt/Windows.UI.Xaml.Automation.Peers.h>
+#include <winrt/Windows.UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.Input.h>
+#include <winrt/Windows.UI.Xaml.Media.h>
+#include <winrt/Windows.UI.Xaml.Shapes.h>
+#include <winrt/Windows.UI.Xaml.h>
```

Comment:
can you include the namespace-agnostic headers instead?
i.e. UI.Composition.h, UI.Xaml.h, etc.

Reply:
I have changed the code code use the namespace-agnostic headers where it is possible.